blocks/library/columns/index.js

```diff
@@ -54,34 +52,20 @@ export const settings = {
 			type: 'number',
 			default: 2,
 		},
-		align: {
-			type: 'string',
-		},
 	},
 
-	description: __( 'A multi-column layout of content.' ),
-
-	getEditWrapperProps( attributes ) {
-		const { align } = attributes;
-
-		return { 'data-align': align };
+	supports: {
+		align: [ 'wide', 'full' ],
 	},
 
+	description: __( 'A multi-column layout of content.' ),
+
 	edit( { attributes, setAttributes, className, focus } ) {
-		const { align, columns } = attributes;
+		const { columns } = attributes;
 		const classes = classnames( className, `has-${ columns }-columns` );
 
 		return [
 			...focus ? [
```

Comment:
@youknowriad should we replace `...focus` with `isSelected`?

Reply:
Definitely

Reply:
Updated 👍